util__deploy_to_maven_central_remove_staged_artifact.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+set -euo pipefail
 
 SSL_PWD=$1
```

Comment:
One more thing, when you will be choosing password which you want to pass as an argument be careful with special characters (at least `$`)

```
$ cat pokus.sh 
#!/bin/bash
set -euo pipefail

SSL_PWD=$1
echo "${SSL_PWD}"
```

```
$ bash pokus.sh "very;$$long pasword"
very;20584long pasword

$ bash pokus.sh "very;$long pasword"
very; pasword
```

Reply:
or be sure you use `' '` as a wrapper

Reply:
Good point indeed. 
Password used has no special characters. If so, this would work, yes:
$ bash takes_1_param.sh 'as$df435'